Clicker/ViewControllers/CardController+Extension.swift

```diff
@@ -31,11 +34,22 @@ extension CardController: ListAdapterDataSource {
     
     func listAdapter(_ listAdapter: ListAdapter, sectionControllerFor object: Any) -> ListSectionController {
         if object is Poll {
-            return PollSectionController()
+            let pollSectionController = PollSectionController()
+            pollSectionController.session = session
+            pollSectionController.userRole = userRole
+            pollSectionController.socket = socket
+            pollSectionController.askedCardDelegate = self
```

Comment:
pass these variables through the init of PollSectionController

Reply:
+1